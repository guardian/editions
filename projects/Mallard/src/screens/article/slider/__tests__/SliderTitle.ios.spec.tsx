import { baseTests } from './SliderTitle-BaseTests'

jest.mock('react-native-device-info', () => ({
    isTablet: () => false,
}))

baseTests('SliderTitle - iOS - Mobile')
