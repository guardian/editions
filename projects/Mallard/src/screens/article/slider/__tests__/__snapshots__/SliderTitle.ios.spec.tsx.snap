// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SliderTitle - iOS - Mobile should display a Front SliderTitle with the correct styles 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "maxWidth": undefined,
      "paddingLeft": 10,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 28,
        }
      }
    >
      Top Stories
    </Text>
    <Text
      style={
        Object {
          "color": "grey",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 28,
        }
      }
    >
       
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "paddingTop": 10,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 5,
          "height": 10,
          "marginRight": 4,
          "width": 10,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 5,
          "height": 10,
          "marginRight": 4,
          "width": 10,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 5,
          "height": 10,
          "marginRight": 4,
          "width": 10,
        }
      }
    />
  </View>
</View>
`;

exports[`SliderTitle - iOS - Mobile should display an Article SliderTitle with a subtitle different to the title 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "maxWidth": 800,
      "paddingLeft": 0,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 20,
        }
      }
    >
      Top Stories
    </Text>
    <Text
      style={
        Object {
          "color": "grey",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 20,
        }
      }
    >
       
      Scary News
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "paddingTop": 10,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
  </View>
</View>
`;

exports[`SliderTitle - iOS - Mobile should display an Article SliderTitle with a subtitle the same as the title 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "maxWidth": 800,
      "paddingLeft": 0,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 20,
        }
      }
    >
      Top Stories
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "paddingTop": 10,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
  </View>
</View>
`;

exports[`SliderTitle - iOS - Mobile should display an Article SliderTitle with the correct styles 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "maxWidth": 800,
      "paddingLeft": 0,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 20,
        }
      }
    >
      Top Stories
    </Text>
    <Text
      style={
        Object {
          "color": "grey",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 20,
        }
      }
    >
       
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "paddingTop": 10,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
  </View>
</View>
`;

exports[`SliderTitle - iOS - Mobile should display an Front SliderTitle with a subtitle different to the title 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "maxWidth": undefined,
      "paddingLeft": 10,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 28,
        }
      }
    >
      Top Stories
    </Text>
    <Text
      style={
        Object {
          "color": "grey",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 28,
        }
      }
    >
       
      Scary News
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "paddingTop": 10,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 5,
          "height": 10,
          "marginRight": 4,
          "width": 10,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 5,
          "height": 10,
          "marginRight": 4,
          "width": 10,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 5,
          "height": 10,
          "marginRight": 4,
          "width": 10,
        }
      }
    />
  </View>
</View>
`;

exports[`SliderTitle - iOS - Mobile should display an Front SliderTitle with a subtitle the same as the title 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "maxWidth": undefined,
      "paddingLeft": 10,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 28,
        }
      }
    >
      Top Stories
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "paddingTop": 10,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 5,
          "height": 10,
          "marginRight": 4,
          "width": 10,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 5,
          "height": 10,
          "marginRight": 4,
          "width": 10,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 5,
          "height": 10,
          "marginRight": 4,
          "width": 10,
        }
      }
    />
  </View>
</View>
`;

exports[`SliderTitle - iOS - Mobile should display an default SliderTitle with a prescribed start index 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "maxWidth": 800,
      "paddingLeft": 0,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 20,
        }
      }
    >
      Top Stories
    </Text>
    <Text
      style={
        Object {
          "color": "grey",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 20,
        }
      }
    >
       
    </Text>
  </View>
  <View
    style={
      Object {
        "flexDirection": "row",
        "paddingTop": 10,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(0, 0, 0, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(220, 220, 220, 1)",
          "borderRadius": 4,
          "height": 8,
          "marginRight": 2,
          "width": 8,
        }
      }
    />
  </View>
</View>
`;

exports[`SliderTitle - iOS - Mobile should display no dots when numOfItems is 1 or less 1`] = `
<View
  style={
    Object {
      "alignSelf": "center",
      "maxWidth": 800,
      "paddingLeft": 0,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "flexDirection": "row",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#000000",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 20,
        }
      }
    >
      Top Stories
    </Text>
    <Text
      style={
        Object {
          "color": "grey",
          "fontFamily": "GTGuardianTitlepiece-Bold",
          "fontSize": 20,
        }
      }
    >
       
    </Text>
  </View>
</View>
`;
